presto-docs/src/main/sphinx/security/kerberos-principals.fragment

```diff
@@ -0,0 +1,29 @@
+Kerberos Principals and Keytab Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
There is an identically named section below. Does this replace what's below, or is the for CLI? If for CLI, suggest adding "for CLI" here, and "for the Presto Coordinator" below (or something similar) to differentiate.